### ISSUE TITLE
[FR] enable remaining print time for character type displays

### ISSUE THREAD

**Issue by knickohr** [MarlinFirmware/Marlin#18085]
Hi !

In Marlin 2 I found the remainting print time and status bar calculation. Ist's unfortunately only activated with graphical displays. So why, what*s the difference ?  Is it possible to activate it also for character type displays, like the RipRap 2004 LCD controller ?

I've found a quick and dirty workaround, but I have to break and twist the coding stukture and also exchanging a variable in ultralcd_HD44780.cpp. Burt his is not compatible to your coding.

**Comment by valeriu-ene** [comment]
This feature is already available in Marlin 2.0.7.2, I'm using it since February, with 20x4 LCD. The gcode must has have M73 commands in it.
Check my pictures below, the feature is working great but it has a small defect related to LCD display.
https://github.com/MarlinFirmware/Marlin/issues/21271
(also containing a link to my uploaded config files)

In config file I have enabled: 
SDSUPPORT
REPRAP_DISCOUNT_SMART_CONTROLLER

In config_adv I enabled: 
LCD_SET_PROGRESS_MANUALLY
SHOW_REMAINING_TIME
USE_M73_REMAINING_TIME
ROTATE_PROGRESS_DISPLAY

I hope this helps.
